lib/PDFtoExcel.py

```diff
@@ -100,7 +103,16 @@ def showDestination(self):
     def runAutomation(self):
 
         if self.filename and self.destinationname:  # If we have both a file and destination, run the script with those
-            Automation.format_pdf_to_excel(fname=self.filename, dname=self.destinationname)
+            fp = open(self.filename, 'rb')
```

Comment:
This (and the saving of the workbook below) are the first kinds of things to consider when starting to "layer" the project.

This top layer is focused on getting the user input, getting the file open, then saving the result. So basically saying all input/output is isolated to this layer. Lower layers (the classes/functions that actually do the work) can then focus on the data. This has positive implications in terms of testing, too, that will be seen later.